scripts/lib/samples/test_sample_R8b.cypher

```diff
@@ -0,0 +1,9 @@
+CREAT (n: User {name: 'Barack Obama', 
+                    username: 'BarackObama', 
+                    createdAt: '2007-04-18T20:50:14.000Z', 
+                    followerCount: 113895387, 
+                    followCount: 282270})
+
+MATCH (n: User), (m: User) 
+WHERE n.username = 'BarackObama' and m.username = 'justinbieber'
+CREAT (n)-[r: Follow]->(m)
```

Comment:
does this work? Isn't it `CREATE` not `CREAT`. You can also add `RETURN n;` at the end so you can make a corresponding .out file.